cli/flags.rs

```diff
@@ -1389,6 +1391,18 @@ fn no_check_arg<'a, 'b>() -> Arg<'a, 'b> {
     .help("Skip type checking modules")
 }
 
+fn no_check_certificate_arg<'a, 'b>() -> Arg<'a, 'b> {
+  Arg::with_name("no-check-certificate")
```

Comment:
We need to bike-shed on this flag name, @ry wanted to make it long and obvious that's it is "dangerous"

Reply:
`--accept-insecure-certificates` or `--allow-insecure-certificates` that way it has insecure right in the name and we can see straight up that something bad might be happening.